hashset_bench_test.go

```diff
@@ -0,0 +1,31 @@
+package hashset
+
+import (
+	"math/rand"
+	"testing"
+)
+
+const benchtime = 50000000
```

Comment:
use `-benchmem` instead of a constant, we can init the test map before b.ResetTimer()

Reply:
typo, -benchtime